### PR TITLE
update the normalization in Ahmed body datapipe

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

closes https://github.com/NVIDIA/modulus/issues/143

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: modulus/datapipes/gnn/ahmed_body_dataset.py [hunk 1]
```diff
@@ -461,73 +398,54 @@ def denormalize(self, pred, gt, device) -> Tuple[Tensor, Tensor]:
...
+        # save to file
         save_json(stats, "edge_stats.json")
-
         return stats
 
     def _get_node_stats(self, keys: List[str]) -> Dict[str, Any]:
         """
-        Compute and save the minimum and maximum values of each node attribute
+        Compute and save the mean and standard deviation values of each node attribute
         for the list of keys in the graphs.
```
Comment (NickGeneva) [inline modulus/datapipes/gnn/ahmed_body_dataset.py:449]: And saves statistics to JSON file.

Comment (mnabian) [reply modulus/datapipes/gnn/ahmed_body_dataset.py:449]: Fixed this

File: modulus/datapipes/gnn/ahmed_body_dataset.py [hunk 2]
```diff
@@ -538,42 +456,36 @@ def _get_node_stats(self, keys: List[str]) -> Dict[str, Any]:
...
-                stats[key + "_max"] = torch.maximum(stats[key + "_max"], max_val)
+        for key in keys:
+            stats[key + "_std"] = torch.sqrt(
+                stats[key + "_meansqr"] - torch.square(stats[key + "_mean"])
+            )
+            stats.pop(key + "_meansqr")
 
-        # Save to file
+        # save to file
         save_json(stats, "node_stats.json")
```
[the remark this answers is not in the document]
Comment (mnabian) [reply modulus/datapipes/gnn/ahmed_body_dataset.py:488]: Added an error message. The model package that is uploaded on NGC catalog already includes these json files.

**Review by NickGeneva — APPROVED** [review]
Thanks for edits!

**Review by ktangsali — APPROVED** [review]
LGTM!

**Pull request merged**